tls/s2n_server_new_session_ticket.c

```diff
@@ -32,15 +32,10 @@
 #include "utils/s2n_safety.h"
 #include "utils/s2n_random.h"
 
-#define TLS13_MAX_NEW_SESSION_TICKET_SIZE sizeof(uint8_t)  + /* message type   */ \
-                                          SIZEOF_UINT24    + /* message length */ \
-                                          sizeof(uint32_t) + /* ticket lifetime */ \
-                                          sizeof(uint32_t) + /* ticket age add */ \
-                                          sizeof(uint8_t)  + /* nonce length */ \
-                                          sizeof(uint16_t) + /* nonce */ \
-                                          sizeof(uint16_t) + /* ticket length */ \
-                                          S2N_MAX_TICKET_SIZE_IN_BYTES + /* ticket */ \
-                                          sizeof(uint16_t)   /* extensions length */
+#define TLS13_FIXED_NEW_SESSION_TICKET_SIZE     143
```

Comment:
Does TLS13_FIXED_NEW_SESSION_TICKET_SIZE include the size of the app protocol and the size of the early data context?

Reply:
I think a comment here would help me out a lot